mantidproject/mantid
# Developer documentation for MuonAnalysis

Adds developer documentation for the Muon Analysis interface.

User docs already exist for this interface, explaining how to use it and what the features do. The purpose of these developer docs is different - to explain the design/architecture, especially around the changes to the Data Analysis tab in Mantid 3.8. It briefly covers what the interface is for, how the code is structured and which parts are tested. 

This page is not linked to from anywhere other than the _Category: Interfaces_ page. This is deliberate, to avoid it being confused with the user documentation for this interface.

**To test:**

Ensure the docs build correctly. Look out for Sphinx errors/warnings, missing images or broken links.

No issue number.

No release notes - this documentation is intended for developers, user docs already exist for this interface.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
